pyproject.toml

```
@@ -1,7 +1,15 @@
 [build-system]
 requires = [
+    # NOTE: this needs to be kept in sync with mypy-requirements.txt
     "setuptools >= 40.6.2",
     "wheel >= 0.30.0",
+    "typing_extensions>=3.10",
+    "mypy_extensions>=0.4.3",
+    "typed_ast>=1.4.0,<2; python_version<'3.8'",
+    "tomli>=1.1.0; python_version<'3.11'",
+    "types-psutil",
+    "types-setuptools",
+    "types-typed-ast>=1.5.0,<1.6.0",
```

I bumped this the other day in https://github.com/python/mypy/commit/840a310f0b6c4a4bd7edfb141c8d1731ba4dd027 :)

```suggestion
    "types-typed-ast>=1.5.8,<1.6.0",
```

---

Thanks!